### PR TITLE
Add the path where the UEFI firmware blob stands on Arch Linux

### PR DESCRIPTION
**Pull request by L3n41c**
as shipped with the `ovmf` package.

### CODE REVIEW

File: mkosi [hunk 1]
```diff
@@ -3396,7 +3396,7 @@ def run_qemu(args):
         die("Couldn't find QEMU/KVM binary")
 
     # Look for UEFI firmware blob
-    for firmware in ('/usr/share/edk2/ovmf/OVMF_CODE.fd',):
+    for firmware in ('/usr/share/edk2/ovmf/OVMF_CODE.fd','/usr/share/ovmf/ovmf_code_x64.bin'):
```
Comment (poettering) [inline mkosi]: hmm, that file appears to be appropriate only for x86-64, no? i figure this should be conditionalized appropriately...

Comment (L3n41c) [reply mkosi]: Indeed.
The `ovmf` package comes with two firmware blobs.
I’m now selecting the right one depending on the architecture.

**Comment by lucaswerkmeister** [comment]
Hm, on my system the package includes different paths:
```sh
$ pacman -Ql ovmf
ovmf /usr/
ovmf /usr/share/
ovmf /usr/share/licenses/
ovmf /usr/share/licenses/ovmf/
ovmf /usr/share/licenses/ovmf/License.txt
ovmf /usr/share/ovmf/
ovmf /usr/share/ovmf/x64/
ovmf /usr/share/ovmf/x64/OVMF_CODE.fd
ovmf /usr/share/ovmf/x64/OVMF_VARS.fd
```

**Comment by poettering** [comment]
@lucaswerkmeister can you submit those paths as a PR?

**Comment by lucaswerkmeister** [comment]
@poettering #240

**Pull request merged**